retroactive/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "ogv-retroactive-airdrop",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "@types/lodash": "^4.14.182",
+    "eslint-config-prettier": "^8.5.0",
+    "ethereum-events": "github:AleG94/ethereum-events#8c92ade1d8f489a577399d5d3cd0093ef6c58aa5",
```

Comment:
Note that I had to change this to https://github.com/AleG94/ethereum-events.git#8c92ade1d8f489a577399d5d3cd0093ef6c58aa5 to get it to install for me.